lib/src/make_pop.py

```diff
@@ -0,0 +1,64 @@
+import json
+import re
+import urllib
+
+with open('country-slim3.json') as f:
+  countries = json.load(f)
+
+mappings = {
+  'United Kingdom of Great Britain and Northern Ireland': 'UK',
+  'United States of America': 'US',
+  'Korea, Republic of': 'S Korea',
+  'Korea (Democratic People\'s Republic of)': 'N Korea',
+  'Virgin Islands (U.S.)': 'US Virgin Islands',
+  'Virgin Islands (British)': 'British Virgin Islands',
+  'Lao People\'s Democratic Republic': 'Laos',
+  'Bosnia and Herzegovina': 'Bosnia',
+  'Saint Vincent and the Grenadines': 'Saint Vincent',
+}
+
+def normalize(name):
+  if name in mappings:
+    return mappings[name]
+  # Remove parentheses
+  name = re.sub(' *\([^)]*\)', '', name)
+  # Remove comma qualifier
+  name = re.sub(', .*$', '', name)
+  return name
+
+country_map = {}
+
+for record in countries:
+  country_map[record['alpha-3']] = normalize(record['name'])
+
+response = urllib.urlopen('https://coronadatascraper.com/timeseries-byLocation.json')
+d = response.read()
+
+data = json.loads(d)
+
+pop = {}
+
+present = 0
+absent = 0
+
+for k,v in data.items():
+    if k in country_map:
+        k = country_map[k]
+    k = k.replace(', USA', '')
+    k = k.replace(' County, ', ', ')
```

Comment:
Might want to also check " Parish" and " Borough".  In merging CDS data earlier this week I have a bunch of other messy data cleaning cases.

Reply:
A catalog of CDS spelling differences (and errors) is encoded in this messy cleaning code:
https://github.com/davidbau/covid-19-chart/blob/merged_feeds/lib/load_flat_cds.js

Reply:
OK, this resolves "Parish" and "Borough", so maybe it's usable as a first attempt.

really what we should do is intersect the labels from the two data sources and print a list of "entries without populations" and "populations without matching entries" which would make it much easier to clean up remaining differences.

(or have a single name canonicalization library, but that's annoying to do across languages. I could change the make_pop code to be JS, I guess.